Notebooks/Simulations/simulating_different_telescopes.ipynb

```diff
@@ -90,11 +90,21 @@
    "execution_count": 3,
```

Comment:
<p>since you solved one deprecation warning, do you also want to change the imageio warning? (the gaussian filter I can take care of)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Updated to imageio.v2 by adding:

import imageio.v2 as imageio
from scipy.ndimage import gaussian_filter

and modified:

ngc_data = imageio.imread(ngc_filename, mode='F', pilmode=None)
ngc_conv = gaussian_filter(ngc_square, sigma, mode='nearest', truncate=6)

This runs all cells without warnings in my conda env

Have an excellent day doctor Birrer